packages/components/src/schema/validators/access-and-short-key.ts

```diff
@@ -0,0 +1,8 @@
+import type { AccessKeyPropType, ShortKeyPropType } from '../props';
+
+export const validateAccessAndShortKey = (accessKey?: AccessKeyPropType, shortKey?: ShortKeyPropType) => {
+	if (accessKey && shortKey) {
+		//eslint-disable-next-line no-console
+		console.error('AccessKey and ShortKey are set. AccessKey is prioritized!');
```

Comment:
"AccessKey is prioritized!" würde ich auch nicht schreiben. (Laut Ticket) darf man wirklich nur eine der Props verwenden und alles andere erzeugt einen Fehler.

Reply:
`Log.error` ?!

Reply:
@deleonio `Log.error` oder `throw new Error()`?
Letzteres ist auffälliger und kann auch von Tools wie Sentry getrackt werden.